### PR TITLE
import aliases that can be omitted

### PR DESCRIPTION
**Pull request by aixeshunter**
import aliases that can be omitted, like `kingpin `, `conntrack`, ...

Signed-off-by: aixeshunter <aixeshunter@gmail.com>

### CODE REVIEW

**Comment by brian-brazil** [comment]
I think we should leave these as-is for readability, what Golang is using as the name isn't exactly obvious in these cases.

File: cmd/promtool/main.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ import (
 	"github.com/google/pprof/profile"
 	"github.com/pkg/errors"
 	"github.com/prometheus/client_golang/api"
-	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
```
Comment (beorn7) [inline cmd/promtool/main.go]: I tend to agree with what @brian-brazil said, but this alias can actually go.

Comment (aixeshunter) [reply cmd/promtool/main.go]: I agree it.

**Comment by tariq1890** [comment]
I agree with @brian-brazil. This is simply how `goimports` functions in the latest go runtimes.

**Comment by roidelapluie** [comment]
Can you please rebase this PR? Thanks!

**Comment by bwplotka** [comment]
Sounds like goimports places this behaviour of requiring alias on "v.*" package names as it collides with module versions (https://github.com/golang/go/issues/30051). Closing this then. 

Thanks @aixeshunter anyway!

**Pull request closed without merging**